server/types/Api.ts

```diff
@@ -0,0 +1,35 @@
+import type { IncomingMessage, ServerResponse } from "http"
+import type { KiboApolloClient } from "@kibocommerce/graphql-client"
+import { ApolloQueryResult } from "apollo-client"
+import { FetchResult } from "apollo-link"
+import * as GraphQL from "./GraphQL"
+
+// getGQLClient
+export interface KiboIncomingMessage extends IncomingMessage {
+  _kiboClient: KiboApolloClient
+  searchParams?: string
+  body?: {
```

Comment:
remove the "body" from this interface

Reply:
As we are passing an Product object from body, either we need body property in KiboApolloClient interface or we need to define **req: any**  to resolve the typescript issue. I think instead of using any keyword we should keep the body property in KiboApolloClient interface. 

Let me know if you think otherwise.
 
![image](https://user-images.githubusercontent.com/89385472/140012206-44171d28-57cd-43b7-8c9d-2db3ea58499f.png)